packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFGeneratedWebKitApis.h

```diff
@@ -342,10 +342,10 @@ typedef NS_ENUM(NSUInteger, FWFWKMediaCaptureType) {
 - (instancetype)init NS_UNAVAILABLE;
 + (instancetype)makeWithCode:(NSNumber *)code
                       domain:(NSString *)domain
-        localizedDescription:(NSString *)localizedDescription;
+                    userInfo:(nullable NSDictionary<NSString *, id> *)userInfo;
 @property(nonatomic, strong) NSNumber *code;
 @property(nonatomic, copy) NSString *domain;
-@property(nonatomic, copy) NSString *localizedDescription;
+@property(nonatomic, strong, nullable) NSDictionary<NSString *, id> *userInfo;
```

Comment:
Collections should be `copy`.

Reply:
I don't understand what you mean by this. This is generated code. Should this be a change to `pigeon`?